gym/vector/__init__.py

```diff
@@ -39,7 +45,14 @@ def make(id, num_envs=1, asynchronous=True, **kwargs):
     """
     from gym.envs import make as make_
     def _make_env():
-        return make_(id, **kwargs)
+        env = make_(id, **kwargs)
+        if wrappers is not None:
+            if callable(wrappers):
+                env = wrappers(env)
+            elif isinstance(wrappers, Iterable) and all([callable(w) for w in wrappers]):
```

Comment:
looks like `collections.abc.Iterable` is not available in python 2.7 (and either way I think silently ignoring the argument if it is not an iterable or callable is not great, let's just try iterating and let python throw an error if it is not possible

Reply:
(that's what fails the travis test)

Reply:
@pzhokhov I've got a work-around import for both Python 2 and 3. And when it's neither callable or Iterable, a NotImplementedError is raised, because we do want the wrappers to be either a single one or a sequence of wrappers that would apply on top of the environment.